### DISCUSSION TITLE
What 'nationZonesSize:1' is use for?

### DISCUSSION THREAD

**Discussion by ProblemsSender** [TownyAdvanced/Towny#5185] [Q&A] [answered]
I don't understand this line in config.yml files, can someone clarify it for me? How is it used?

**Comment by rjs06** [comment, marked as answer]
This is a copy and paste from the config file

Line 732 - 738

  - Nation Zones are a special type of wilderness surrounding Capitals of Nations or Nation Capitals and their Towns.
  - When it is enabled players who are members of the nation can use the wilderness surrounding the town like normal.
 -  Players who are not part of that nation will find themselves unable to break/build/switch/itemuse in this part of the wilderness.
  - The amount of townblocks used for the zone is determined by the size of the nation and configured in the nation levels.
  - Because these zones are still wilderness anyone can claim these townblocks.
  - It is recommended that whatever size you choose, these numbers should be less than the min_plot_distance_from_town_plot otherwise
  - someone might not be able to build/destroy in the wilderness outside their town.

Not sure if this answers your question but here's what I found.

**Reply by LlmDl** [reply]
To add: the 1 would signify a 1 plot ring of nation zone.
